integration-tests/container/benchmark/Dockerfile

```diff
@@ -1,5 +1,13 @@
-FROM phoronix/pts
+FROM --platform=linux/amd64 phoronix/pts as stage-amd64
+FROM --platform=linux/ppc64le quay.io/rhacs-eng/phoronix-pts-ppc64le as stage-ppc64le
```

Comment:
Is this image already available?

Reply:
This image will have to be build once for power.